vuepress/docs/next/docs/getting-started/ent-svc.md

```diff
@@ -1,3 +1,61 @@
 ---
 sidebarDepth: 1
 ---
+
+# Auxiliary Services
+
+The following sections discuss auxiliary services and commands.
+
+## Services
+
+When the Entando Bundle CLI enables a service, a default auxiliary configuration is generated and stored as a Docker Compose YAML file in the bundle's 'svc' folder. Three services are available out of the box: `mysql`, `postgresql` and `keycloak`.
+
+- `mysql` and `postgresql` obey the same naming convention
+   - The username and password are derived from the bundle name, e.g. YOUR-BUNDLE has the username YOUR-BUNDLE-USER and the password YOUR-BUNDLE-PASSWORD
+   - A database is automatically created and given the bundle name, e.g. YOUR-BUNDLE
+
+- `keycloak` contains the Entando-related configuration
+   - The `keycloak` realm name is 'entando-dev'
+   - The console is accessible at *http://localhost:9080* with username: *admin*, password: *admin*
+   - All roles and permissions are preset, based on the Keycloak realm-config file
+   - The `keycloak.yaml`, realm-config and user config files are located in the 'keycloak' folder inside 'svc'
+
+A custom service can be added to a bundle's 'svc' folder as a Docker Compose YAML file. The file and service must have the same name, e.g. if the service filename is `mysql.yml`, then the service name must be `mysql`:
```

Comment:
a custom service can be added 'as' a Docker Compose... >> as sounds odd, maybe with?

Reply:
the service is the file